drivers/led/gpio.c

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <kernel.h>
+#include <led.h>
+#include <gpio.h>
+#include <device.h>
```

Comment:
Sort includes accordingly

Reply:
I put this in alphabetical order. Is this what you mean ?

Reply:
yeah